services/user/user.go

```diff
@@ -77,7 +77,7 @@ func RenameUser(ctx context.Context, u *user_model.User, newUserName string) err
 		}
 	}
 
-	if err = repo_model.UpdateRepositoryOwnerName(ctx, oldUserName, newUserName); err != nil {
+	if err = repo_model.UpdateRepositoryOwnerNames(u.ID, newUserName); err != nil {
```

Comment:
>  UpdateRepositoryOwnerNames updates repository owner_names (this should only be used when the ownerName has changed case)

As the comment said, this function should only be used when the ownerName has changed case, this function should only be used in L61 but not here?
Not clearly sure about the difference between this two function.